boards/xtensa/nxp_adsp_imx8x/Kconfig.defconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NXP_ADSP_IMX8X
+
+config BOARD
+	default "nxp_adsp_imx8x"
+
+config DUMMY_DMA
+	default y
+	depends on DMA
```

Comment:
what's the purpose of this ?

I can't find it's usage or description on where this exists in Kconfig or why.

Reply:
This is the software DMA driver used by i.MX and is defined in SOF - https://github.com/zephyrproject-rtos/sof/blob/zephyr/src/drivers/Kconfig#L11. 
We need this for the communication between the host and DSP.

Reply:
thanks.